### ISSUE TITLE
Auto-Beat[HR]

### ISSUE THREAD

**Issue by munchyluvsphex** [Prodigy-Hacking/ProdigyMathGameHacking#666]
**Describe the hack in 5 words or less:**

I know you can get all of the key items already but it wont let you put it on the peddle stools, so what it should do is go to every place and do all of the challenges but im not sure how do it, i am kind of experience in Lua and I know you can do pathfinding but I'm not sure how that will work thank you!
**Have you made sure this hack isn't available yet?* (Yes/No)**

**Comment by TheSchmede** [comment]
> **Describe the hack in 5 words or less:**
> 
> I know you can get all of the key items already but it wont let you put it on the peddle stools, so what it should do is go to every place and do all of the challenges but im not sure how do it, i am kind of experience in Lua and I know you can do pathfinding but I'm not sure how that will work thank you!
> __Have you made sure this hack isn't available yet?_ (Yes/No)_*

That is totally less than 5 words. lol

**Comment by munchyluvsphex** [comment]
yes very much les than five words.

**Comment by munchyluvsphex** [comment]
less*

**Comment by munchyluvsphex** [comment]
@ArcerionDev

**Comment by munchyluvsphex** [comment]
@ChickenOnYT

**Comment by ChickenOnYT** [comment]
> **Describe the hack in 5 words or less:**
> 
> I know you can get all of the key items already but it wont let you put it on the peddle stools, so what it should do is go to every place and do all of the challenges but im not sure how do it, i am kind of experience in Lua and I know you can do pathfinding but I'm not sure how that will work thank you!
> __Have you made sure this hack isn't available yet?_ (Yes/No)_*

sigh

i placed them once, but it doesnt do anything

the ice tower isnt coded and all that

if it was i would've leaked it on the pmgw by now

**Comment by munchyluvsphex** [comment]
thank you!

**Comment by ChickenOnYT** [comment]
np

**Comment by NX721** [comment]
...

**Comment by ArcerionDev** [comment]
> **Describe the hack in 5 words or less:**
> 
> I know you can get all of the key items already but it wont let you put it on the peddle stools, so what it should do is go to every place and do all of the challenges but im not sure how do it, i am kind of experience in Lua and I know you can do pathfinding but I'm not sure how that will work thank you!
> __Have you made sure this hack isn't available yet?_ (Yes/No)_*

yeah prodigy isn't coded in lua and @ChickenOnYT is right, there's no way to actually get it as of now

**Comment by ChickenOnYT** [comment]
mhm

**Comment by ArcerionDev** [comment]
ok so die issue
